docker-compose.yml

```
@@ -194,3 +195,21 @@ services:
     working_dir: /home/terraform
     volumes:
       - ./provisioning:/home/terraform
+
+  dashboard:
+    container_name: dashboard
+    build:
+      context: ./src/dashboard
+      args:
+        DOCKER_UID: ${DOCKER_UID:-1000}
+        DOCKER_GID: ${DOCKER_GID:-1000}
+    user: ${DOCKER_USER:-1000}
+    ports:
+      - "8030:8030"
```

You can use the default Django port and adapt the mapping:

```suggestion
      - "8030:8000"
```